examples/time_series/goes_xrs_nrt_data.py

```diff
@@ -64,9 +67,30 @@
 goes_ts = ts.TimeSeries(goes_data, meta, units, source="xrs")
 
 ###############################################################################
-# Finally, we can plot the timeseries.
+# Now we will download the latest flare event information
+
+flare_events = pd.read_json('https://services.swpc.noaa.gov/json/goes/primary/xray-flares-7-day.json')
+
+###############################################################################
+# Next we load it into an astropy Table
+
+goes_class = [str(this_class) for this_class in flare_events['max_class'].values]
+start_time = parse_time(flare_events['begin_time'].values)
+end_time = parse_time(flare_events['end_time'].values)
+peak_time = parse_time(flare_events['max_time'].values)
+flare_list = Table(data={"class": goes_class, "start_time": start_time, "peak_time": peak_time, "end_time": end_time})
```

Comment:
Table has a `.frompandas` or something like that, did you try it?

Reply:
That does work though I still have to do the work to convert the times so it really just saves one line of code.

Reply:
That is unfortunate.